paddlenlp/transformers/xlnet/tokenizer.py

```diff
@@ -271,7 +281,7 @@ def convert_ids_to_tokens(self, ids, skip_special_tokens=False):
         return tokens
 
     def convert_tokens_to_string(self, tokens):
```

Comment:
There seems to be a bug when generating the doc:
":param tokens: List of tokens. :type tokens: list(str)"

Reply:
this doctoring is inherited from superclass, need to modify superclass's docs instead